packages/fluentui/react-northstar/src/components/Menu/MenuItem.tsx

```diff
@@ -215,6 +237,17 @@ export const MenuItem = compose<'a', MenuItemProps, MenuItemStylesProps, {}, {}>
 
     const slotProps = composeOptions.resolveSlotProps<MenuItemProps & MenuItemState>({
       ...props,
+      accessibility,
```

Comment:
It will be hard to follow what props should be defined here in the future, which may lead to unexpected bugs. Can we refactor this to something like:
```
...props,
...parentProps,
accessibility,
variables: mergeComponentVariables(variables, parentProps.variables),
isFromKeyboard,
menuOpen,
```

Reply:
Done